albumentations/augmentations/domain_adaptation.py

```diff
@@ -1,16 +1,28 @@
-from typing import List
+from typing import List, Union
 import random
 
 import cv2
 import numpy as np
 
 from ..core.transforms_interface import ImageOnlyTransform, to_tuple
-from .functional import clipped
+from .functional import clipped, preserve_shape
 
-__all__ = ["HistogramMatching", "FDA", "fourier_domain_adaptation"]
+from skimage.exposure import match_histograms
+
+__all__ = ["HistogramMatching", "FDA", "fourier_domain_adaptation", "read_bgr_image", "read_rgb_image"]
+
+
+def read_bgr_image(path):
+    return cv2.imread(path, cv2.IMREAD_COLOR)
+
+
+def read_rgb_image(path):
```

Comment:
Maybe we could move those 2 functions to another module with generic functions such as `albumentations.augmentations.utils` since those functions are not tied to transforms in `domain_adaptation.py`?

Reply:
done